frame/balances/README.md

```diff
@@ -53,16 +53,16 @@ locks always operate over the same funds, so they "overlay" rather than "stack".
 The Balances module provides implementations for the following traits. If these traits provide the functionality
 that you need, then you can avoid coupling with the Balances module.
 
-- [`Currency`](../frame_support/traits/trait.Currency.html): Functions for dealing with a
+- [`Currency`](https://docs.rs/frame-support/traits/trait.Currency.html): Functions for dealing with a
 fungible assets system.
-- [`ReservableCurrency`](../frame_support/traits/trait.ReservableCurrency.html):
+- [`ReservableCurrency`](https://docs.rs/frame-support/traits/trait.ReservableCurrency.html):
```

Comment:
so, it looks like we have a slight issue here - this gives me a 404 .. the link would have to be
`https://docs.rs/frame-support/latest/frame_support/traits/trait.ReservableCurrency.html`

I am  not sure how the rewrite currently does it, but external links (with `../`) are not done correctly (for `docs.rs`)

Reply:
Ok, will fix it asap.